packages/pwa/app/pages/cart/partials/cart-secondary-button-group.jsx

```diff
@@ -23,12 +23,22 @@ import {noop} from '../../../utils/utils'
 import useCustomer from '../../../commerce-api/hooks/useCustomer'
 
 export const REMOVE_CART_ITEM_CONFIRMATION_DIALOG_CONFIG = {
```

Comment:
Just check the code, we have two places that declared `REMOVE_CART_ITEM_CONFIRMATION_DIALOG_CONFIG` (one here, and another at `pages/account/constant.js`)? Can we remove one? 🤔

Reply:
Thanks, Alex. I've removed one of them.